pallets/attestation/src/mock.rs

```diff
@@ -165,12 +167,23 @@ pub fn insert_attestation<T: Config>(claim_hash: ClaimHashOf<T>, details: Attest
 	}
 }
 
+pub fn sr25519_did_from_seed(seed: &[u8; 32]) -> SubjectId {
+	MultiSigner::from(sr25519::Pair::from_seed(seed).public())
```

Comment:
I needed the function in my `pallet-migration` tests and I wanted to avoid code duplication. The functions are behind the test feature and I could not access them.